src/index.js

```diff
@@ -0,0 +1,102 @@
+const fs = require('fs');
+const { git, gitMultilineResults, toTitleCase } = require('./utils');
+
+let packageName = JSON.parse(fs.readFileSync('./package.json').toString()).name;
+
+const packageNameAliases = {
+	'balena.io': 'resin-api',
+};
+
+packageName = packageNameAliases[packageName] || packageName;
+
+const moduleAliases = {
+	'resin-ui': 'dashboard',
+};
+
+const moduleName =
+	moduleAliases[packageName] ||
+	packageName.replace('resin-', '').replace('balena-', '');
+
+function getLatestProdInfo() {
+	const tags = gitMultilineResults('tag -l --sort=refname production-*');
+	let latestProdTag = tags[tags.length - 1];
```

Comment:
This will throw an error if there are no `production-*` tags, but I guess its not a big deal if we consider the purpose of the script.

Reply:
will this also match if the tag is simply `production`?